src/index.css

```diff
@@ -14,26 +14,57 @@
   -webkit-text-size-adjust: 100%;
 }
 
-a {
-  font-weight: 500;
-  color: #646cff;
-  text-decoration: inherit;
-}
-a:hover {
-  color: #535bf2;
-}
-
 body {
   margin: 0;
   display: flex;
-  place-items: center;
-  min-width: 320px;
+  align-items: center;
+  justify-content: center;
+  min-width: 500px;
   min-height: 100vh;
+  background-color: #eecbc7; 
+  background-image: url("public/iridescent.jpeg");
+  color: #9c8399; 
+}
+
+.container {
+  display: flex;
+  flex-direction: column; 
+  justify-content: center;
+  align-items: center;
+  background-color: #f6dfd7; 
+  border-radius: 10px;  
+  width: 1150px; 
+  height: 300px;
 }
```

Comment:
![image](https://github.com/bpb2008/MagicQuotes/assets/152167/7736a1dc-636c-498b-8b06-fa1dad5438fd)
See what's wrong here? Can you get the text to wrap down to the next line and for the width of the container to be more dynamic for narrower screen sizes? (Can open the Chrome devtools panel and resize the window to see how it shows for different screen widths.)

Reply:
So I mostly solved this, but now I'm trying to figure out how to keep the container as a consistent size!